apps/web/app/lang/de.json

```diff
@@ -272,23 +205,37 @@
       "success": "Ihr Konto wurde erfolgreich erstellt."
     }
   },
-  "averageRating": "Durchschnittliche Bewertung",
-  "back": "Zurück",
-  "backToCheckout": "Zurück zur Kasse",
-  "backToShopping": "Zurück zum Einkaufen",
-  "basePrice": "Grundpreis",
-  "basedOnratings": "Basierend auf {count} Bewertung(en)",
+  "availabilitiesOptions": {
+    "availabilityBackOrder": "Lieferrückstand (BackOrder)",
+    "availabilityDiscontinued": "Abgekündigt (Discontinued)",
+    "availabilityInStock": "Auf Lager (InStock)",
+    "availabilityInStoreOnly": "Nur im Ladengeschäft verfügbar (InStoreOnly)",
+    "availabilityLimitedAvailability": "Begrenzt verfügbar (LimitedAvailability)",
+    "availabilityNotSet": "Verfügbarkeit wählen",
+    "availabilityOnlineOnly": "Nur online verfügbar (OnlineOnly)",
+    "availabilityOutOfStock": "Nicht auf Lager (OutOfStock)",
+    "availabilityPreOrder": "Vorbestellbar (PreOrder)",
+    "availabilityPreSale": "Vorverkauf (PreSale)",
+    "availabilitySoldOut": "Ausverkauft (SoldOut)"
+  },
   "billing": {
     "addButton": "Rechnungsadresse hinzufügen",
     "description": "Fügen Sie eine Rechnungsadresse hinzu. Sie werden die Rechnung an die oben angegebene E-Mail-Adresse erhalten.",
     "heading": "Rechnungsadresse",
     "methodChanged": "Die von Ihnen ausgewählte Zahlungsart ist nicht mehr verfügbar."
   },
-  "billingAddressRequired": "Bitte Rechnungsadresse auswählen.",
-  "breadcrumbsDropdownText": "Mehr Breadcrumbs anzeigen",
-  "buy": "Kaufen",
-  "cancellationRights": "Widerrufsrecht",
-  "cart": "Warenkorb",
+  "cart": {
+    "empty": "Ihr Warenkorb ist leer",
+    "emptyImageAlt": "Bild eines leeren Warenkorbs",
+    "emptyNotification": "Sie haben keine Artikel in Ihrem Warenkorb",
+    "emptyWishlist": "Ihre Wunschliste ist leer",
+    "itemAdded": "Artikel zum Warenkorb hinzugefügt",
+    "itemCount": "(Artikel: {count})",
+    "itemRemoved": "Artikel aus dem Warenkorb entfernt",
+    "myCart": "Mein Warenkorb",
+    "numberInCart": "{count} Artikel im Warenkorb",
+    "numberInWishlist": "{count} Artikel in der Wunschliste"
+  },
   "categories": {
```

Comment:
Please cleanup this whole category and then move it somewhere else. It has lots of unused keys. If its really all just footer stuff, either put it under homepage.footer or just under footer